src/main/java/com/github/vkorobkov/jfixtures/instructions/CleanTable.java

```diff
@@ -1,15 +1,17 @@
 package com.github.vkorobkov.jfixtures.instructions;
 
+import com.github.vkorobkov.jfixtures.config.structure.tables.CleanMethod;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 
 @AllArgsConstructor
+@Getter
 public class CleanTable implements Instruction {
-    @Getter
     private final String table;
+    private final CleanMethod cleanMethod;
 
     @Override
     public void accept(InstructionVisitor visitor) {
-        visitor.visit(this);
+        visitor.visit(this, this.cleanMethod);
```

Comment:
If you pass `this` - why do you need to pass `this.cleanMethod`?

Reply:
:-1: my bad